site/content/en/docs/tasks/integrate_a_custom_job.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Integrate a custom Job"
+date: 2022-07-25
+weight: 8
+description: >
+  Integrate a custom Job with Kueue.
+---
+# Integrate a custom Job with Kueue
+
+This page shows how to integrate a custom Job with Kueue. 
+
+## What we need to build
+
+1. Configration
+2. Controller
+3. Webhook
```

Comment:
nit: it might be worth mentioning to say that the build system needs to be adjusted. The Makefile and the dependency.